src/components/Navbar/SideMenu.tsx

```diff
@@ -0,0 +1,99 @@
+"use client";
+import { RxHamburgerMenu } from "react-icons/rx";
+import { IoCloseOutline } from "react-icons/io5";
+import { useEffect, useState } from "react";
+import { motion, AnimatePresence } from "framer-motion";
+import { FaPlus } from "react-icons/fa6";
+const SideMenu = () => {
+  const [openNavigationMenu, setOpenNavigationMenu] = useState<boolean>(false);
```

Comment:
@Rauniyar-shovit please rename this variable to isNavMenuOpen

const [isNavMenuOpen, setIsNavMenuOpen] = useState<boolean>(false);
It's important to name variable names correctly, where it makes sense